data/org.freedesktop.portal.Screenshot.xml

```diff
@@ -29,7 +29,7 @@
       via the document portal, and the returned URI will point
       into the document portal fuse filesystem in /run/user/$UID/doc/.
 
-      This documentation describes version 2 of this interface.
+      This documentation describes version 3 of this interface.
```

Comment:
What's the purpose of this version bump?

Reply:
It's documented in the commit message: 

> Portal implementations will need to be adapted to this change,
> even though no actual D-Bus API change is introduced. Basically
> they'll have to skip showing any kind of dialog if the screenshot
> portal version is equal or greater than 3.

In other words, even if the D-Bus API is intact, v3 of the portal asks for the `screenshot` permissions, v2 doesn't, and implementations need to know this to act